### ISSUE TITLE
luautils-0.1.3 does not compile

### ISSUE THREAD

**Issue by peti** [ajnsit/luautils#3]
Citing from http://hydra.cryp.to/build/945672/nixlog/1/raw:

~~~~
Building luautils-0.1.3...
Preprocessing library luautils-0.1.3...

src/Scripting/LuaUtils.hs:2:35: Warning:
    -XOverlappingInstances is deprecated: instead use per-instance pragmas OVERLAPPING/OVERLAPPABLE/OVERLAPS
[1 of 1] Compiling Scripting.LuaUtils ( src/Scripting/LuaUtils.hs, dist/build/Scripting/LuaUtils.o )

src/Scripting/LuaUtils.hs:87:10:
    Duplicate instance declarations:
      instance [overlap ok] Lua.StackValue a => Lua.StackValue [a]
        -- Defined at src/Scripting/LuaUtils.hs:87:10
      instance Lua.StackValue a => Lua.StackValue [a]
        -- Defined in ‘Scripting.Lua’
~~~~

I suppose the code can't cope with recent versions of hslua, i.e. version 0.4.0?

**Comment by ajnsit** [comment]
Yep, I will take a look.

**Comment by ajnsit** [comment]
Things changed a bit in HsLua 0.4.0. This should be fixed with (8ce202c7241894d03946e32cfb21029f953000d4). I will push v0.1.4 to Hackage soon.

**Comment by ajnsit** [comment]
Pushed v0.1.4 to hackage
